pom.xml

```diff
@@ -74,6 +77,62 @@
       <version>${sonar.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>${version.javax.servlet}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jvnet.com4j</groupId>
+      <artifactId>com4j</artifactId>
+      <version>${version.com4j}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jvnet.com4j.typelibs</groupId>
+      <artifactId>active-directory</artifactId>
+      <version>${version.org.jvnet.com4j.typelibs}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jvnet.com4j.typelibs</groupId>
+      <artifactId>ado20</artifactId>
+      <version>${version.org.jvnet.com4j.typelibs}</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
+      <version>2.6</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>18.0</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-collections</groupId>
+      <artifactId>commons-collections</artifactId>
+      <version>3.2.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+      <version>3.0.0</version>
```

Comment:
jsr305 should be declared with scope `provided`, as it's used only at compilation time.